frontend/js/src/common/listens/AddToPlaylist.tsx

```diff
@@ -100,7 +102,7 @@ export default NiceModal.create((props: AddToPlaylistProps) => {
               message={
                 <>
                   Successfully added <i>{trackToSend.title}</i> to playlist{" "}
-                  <a href={playlistIdentifier}>{playlistName}</a>
+                  <Link to={playlistUrl.pathname}>{playlistName}</Link>
```

Comment:
I think you should be able to set the identifier directly as it is already a fully qualified URL:
```suggestion
                  <Link to={playlistIdentifier}>{playlistName}</Link>
```
Of course that means we won't need the new URL at line 92.
I will test after the change.

Reply:
Oh ok! I actually introduced that in the second commit because I wasn't sure how `react-router` deals with full URLs and couldn't find anything in the docs. I pushed the changes you suggested, thanks!